app/controllers/api/plans_base_controller.rb

```diff
@@ -43,7 +43,8 @@ def find_plan
 
   def find_plans
     search = ThreeScale::Search.new(params[:search] || params)
-    @plans = collection.order_by(params[:sort], params[:direction])
+    @plans = collection.not_custom
```

Comment:
Can collection be:

```ruby
def collection
  super.not_custom
end
```

So it applies everywhere in this controller?

Reply:
OK but it's that what's supposed to be? Always `not_custom`?